Makefile.config.in

```diff
@@ -225,6 +225,7 @@ WITH_DEBUGGER=@with_debugger@
 WITH_CAMLTEX=@with_camltex@
 WITH_OCAMLDOC=@ocamldoc@
 WITH_OCAMLTEST=@ocamltest@
+DOCUMENTATION_TOOL=@documentation_tool@
```

Comment:
```suggestion
DOCUMENTATION_TOOL=@documentation_tool@
DOCUMENTATION_TOOL_CMD=@documentation_tool_cmd@
```
Regardless of this change, please could this go in `Makefile.build_config` - the (slow) aim is that `Makefile.config` (which is installed) should only contain things which may be of interest post-installation.

Reply:
I have moved those variables to `Makefile.build_config`, thanks for the suggestion.